src/sdp/DefaultSDP.ts

```diff
@@ -619,4 +643,36 @@ export default class DefaultSDP implements SDP {
     const newSDP = sections.join('');
     return new DefaultSDP(newSDP);
   }
+
+  mediaSections(): SDPMediaSection[] {
+    const sections = DefaultSDP.splitSections(this.sdp);
+    if (sections.length < 2) {
+      return [];
+    }
+
+    const parsedSections: SDPMediaSection[] = [];
+    for (let i = 1; i < sections.length; i++) {
+      const section = new SDPMediaSection();
+      const lines = DefaultSDP.splitLines(sections[i]);
+      for (const line of lines) {
+        const mediaType = DefaultSDP.mediaType(line);
+        if (mediaType) {
```

Comment:
If it is not media, should we just break from this? Any reason to continue?

Reply:
There is no sections that are not media sections after they begin, and we start from index 1 to avoid that non-media section.